app/components/find/courses/about_schools_component/view.rb

```diff
@@ -7,16 +7,17 @@ class View < ViewComponent::Base
         include PublishHelper
         include PreviewHelper
 
-        attr_reader :course
+        attr_reader :course, :preview
 
         delegate :published_how_school_placements_work,
                  :program_type,
                  :study_sites,
                  :site_statuses, to: :course
 
-        def initialize(course)
+        def initialize(course, preview:)
```

Comment:
```suggestion
        def initialize(course, preview: false)
```
This may stop make older uses of this Component from having to be updated